client/src/app/app-routing.module.ts

```diff
@@ -1,10 +1,16 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { NotFoundComponent } from './not-found/not-found.component';
 
-const routes: Routes = [];
+const routes: Routes = [
```

Comment:
Don't you have any routes?

Reply:
Yeah I modified this file and use lazy loading